meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -12,6 +12,9 @@ python __anonymous() {
         d.appendVar('DEPENDS', ' virtual/kernel')
         if 'modsign' in d.getVar('DISTRO_FEATURES'):
             d.setVar('INHERIT_KERNEL_MODSIGN', 'kernel-modsign')
+
+    if bb.data.inherits_class('go-mod', d):
+            d.appendVarFlag('do_compile', 'network', '1')
```

Comment:
Wonder if the lack of proxy will cause the older issue to happen again, will do a local test.

Reply:
the use of an invalid proxy cause a build fail https://github.com/foundriesio/meta-lmp/pull/824#issuecomment-1263801790 but I don't remember to did the test without proxy. so I will do that as well

Reply:
Tested without proxy https://github.com/foundriesio/meta-lmp/pull/824#issuecomment-1276476242 and works in the same way.

Reply:
Cool, thanks!

Just noticed there is 4 extra spaces here, can you fix to 4 to be aligned with the lines above it?

Reply:
Fixed, I don't even know how python didn't complain about the indentation!

Reply:
I guess it would complain if it was a tab instead.